diktat-rules/src/test/resources/test/paragraph2/kdoc/ConstructorCommentNoKDocTest.kt

```diff
@@ -5,7 +5,7 @@ class A constructor(
 ) {}
 
 class A constructor(
-    //single-line comment
+    // single-line comment
```

Comment:
why do we change a test?

Reply:
Just for the sake of cosmetic beauty. This test has nothing to do with checking this indentation.